TESTS/netsocket/tls/main.cpp

```diff
@@ -196,7 +196,10 @@ Case cases[] = {
     Case("TLSSOCKET_SEND_TIMEOUT", TLSSOCKET_SEND_TIMEOUT),
     Case("TLSSOCKET_NO_CERT", TLSSOCKET_NO_CERT),
 #ifndef __IAR_SYSTEMS_ICC__
```

Comment:
I think we should remove the `#ifndef __IAR` as well.

Reply:
Good point. I removed it, but added a further warning comment, so someone doesn't spend hours debugging this test in case it gets uncommented in distant future.